proto/decentraland/social_service/v2/social_service_v2.proto

```diff
@@ -95,20 +102,31 @@ message UpsertFriendshipResponse {
 }
 
 message FriendshipUpdate {
+  message RequestResponse {
```

Comment:
We also need the profile information of the involved users. I think we should create a message `FriendRequest` which includes all information of a friend request:
- FriendProfile friend
- int64 created_at
- optional string message
- string id = 4;